addon/pods/components/rui-error-template/template.hbs

```diff
@@ -14,5 +14,10 @@
     <div class="rui-error-template-tagline">
       {{tagline}}
     </div>
+    {{#if link}}
+      <a class='btn btn-secondary btn-lg rui-error-button' href={{link}}>
```

Comment:
this is still not a good setup for having internal links because it will force the reload of the ember app even if it's internal.